src/mount/mod.rs

```diff
@@ -5,8 +5,14 @@ mod linux;
 #[cfg(linux_android)]
 pub use self::linux::*;
 
-#[cfg(bsd)]
+#[cfg(bsd_without_apple)]
 mod bsd;
```

Comment:
We may want to change module and file name as well, for the sake of consistency:)

```suggestion
mod bsd_without_apple;
```